src/domain/occurrence/utils.ts

```diff
@@ -23,7 +23,13 @@ export const hasOccurrenceSpace = (
 
 export const getAmountOfSeatsLeft = (
   occurrence: OccurrenceFieldsFragment
-): number => occurrence.amountOfSeats - (occurrence.seatsTaken || 0);
+): number => {
+  const amountOfSeatsLeft =
+    occurrence.amountOfSeats - (occurrence.seatsTaken || 0);
+
+  // Don't return negative space left, but a 0 instead
+  return amountOfSeatsLeft < 0 ? 0 : amountOfSeatsLeft;
```

Comment:
might be as well Math.max(0, amountOfSeatsLeft) ?

Reply:
good point 👍

Reply:
I agree this looks better. Needed to check what internet says about performance: https://stackoverflow.com/q/1232345/784642. I think I update it.

Reply:
Maybe if were executing this thousands of times in a second regularly we should consider perf implications but not now when it is basically called couple of times :)

Reply:
<img width="591" alt="Screenshot 2021-02-25 at 11 04 22" src="https://user-images.githubusercontent.com/15219142/109129380-434f5780-7759-11eb-943b-c173588ad76b.png">
Maybe not even then. Executing that 100 million times take under a second on my computer...

Reply:
well that is acutally 1 billion times :D?

Reply:
I was just curious 😂